src/components/Message/views/Notification/Notification.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC, ReactElement } from 'react';
+import { defaultIconFromType } from './utils';
+import {
+    StyledNotification,
+    StyledNotificationMainContainer,
+    StyledNotificationIconContainer,
+    StyledNotificationTextContainer,
+} from './styles';
+import { InnerRef } from '../../../../types/utils/InnerRef';
+import { Text } from '../../../Typography';
+import { MessageProps } from '../../types';
+import { Link } from '../../../Typography/Link';
+
+export type NotificationViewProps = {
+    Icon?: ReactElement;
+    description?: String;
+    link?: String;
+} & InnerRef<MessageProps>;
+
+export type NotificationViewType = FC<NotificationViewProps>;
+
+export const NotificationView: NotificationViewType = ({
+    children,
+    className,
+    type = 'default',
+    innerRef,
+    Icon = defaultIconFromType(type),
+    description,
+    link,
+    ...ContainerProps
+}) => {
+    return (
+        <StyledNotification
+            className={className}
+            ref={innerRef}
+            type={type}
+            {...ContainerProps}
```

Comment:
`className` se ti rozbaluje v tom `...containerProps`, tzn. tady můžeš to explicitní předávání className smazat a nechat jen `ref`, `type`, `...containerProps`.